azavea/district-builder-dtl-pa
# Small fixes and changes for C70

## Overview

 * Remove password number and uppercase letter requirements 
 * Change map name on sidebar to "Statewide Stats"
 * Change youth division label to "Age 13 through Grade 12"
 * Fix selected basemap styling on basemap picker
 * Point "Support" link on sign in to user guide
 * Add 'Non-PA Resident' to county on contest form

### Checklist


## Testing Instructions

 * Run migrations for the youth division part of this PR. 
 * Try to create a new account, make sure that password length requirements still exist, that the other requirements don't, and that the error message on weak password has been changed.
 * Log in, open a map, and see that the sidebar has the phrase "Statewide Stats" in place of the map name.
 * Check that the basemap picker works and that the styling for the current basemap is good.
 * Check that the support link after log in on the header points to the support PDF. 
 * Submit a plan and in the form, make sure that the youth division has the correct text. Then generate a submission report and check that the text there is correct. 
 * On the contest submission form, also check to see if 'Non-PA Resident' is a county option. Select it and check the report to see the region also be 'Non-PA Resident'.

Closes [#161304162](https://www.pivotaltracker.com/story/show/161304162), [#161304021](https://www.pivotaltracker.com/story/show/161304021), [#161304092](https://www.pivotaltracker.com/story/show/161304092), [#161266050](https://www.pivotaltracker.com/story/show/161266050), [#161266010](https://www.pivotaltracker.com/story/show/161266010), [#161266152](https://www.pivotaltracker.com/story/show/161266152)
